wix/main.wxs

```diff
@@ -353,7 +355,7 @@
             <ComponentRef Id='binary20'/>
             <ComponentRef Id='binary21'/> -->
             <ComponentRef Id='binary22'/>
-            <ComponentRef Id='binary23'/>
```

Comment:
Makes sense to me

Reply:
I saw this earlier today and immediately sighed ugh! I went through the wix changes and I think they look right. Hopefully nothing pukes. Let me rephrase, winget is nearly guaranteed to puke every release, let's hope none of these changes cause winget to puke.

Reply:
Yeah, last time I just didn't realize that the binaryXX had to be referenced somewhere else too. This time I know about it 😉 so I've just copied examples of others being removed and that should hopefully be okay.

I have a VM that almost has everything set up, so I can verify if I can actually get the wix stuff to work (last time it choked somehow)

Reply:
Sounds good. You probably don't know the history but it's not an exaggeration to say that our winget PR fails 95% of the time. 99.99% of the time it's because of wingets CI which is a royal pain and doesn't work very well. Other times is when we've made mistakes like the one you referenced. I can deal with people making mistakes every now and then. What bothers me is the frequency that nushell fails on winget that it's not really anyone's fault other than the authors of the winget ci.

Reply:
Ouch. Well, good to know.

Can't help but feel that even if the windows installer XML is a legacy thing, they should build an optional config on top of it that's nicer to use... Leaky XML serialization sucks as an interface